README.md

```diff
@@ -8,11 +8,11 @@ Always strive for specificity over brevity. The more specific our class selector
 
 ### Syntax
 
-We use the SCSS flavor of SASS to preprocess our CSS.
+We use the SCSS flavor of SASS to preprocess our CSS. You might want to use [scss-lint](https://github.com/brigade/scss-lint) with the configuration provided in this repository.
```

Comment:
Hey @lozandier, thanks for the suggestion! Frankly, I don't even know if this PR is still valid for what has been done.